addons/Popochiu/Editor/MainDock/TabRoom.gd

```diff
@@ -95,14 +130,48 @@ func scene_changed(scene_root: Node) -> void:
 		
 		_room_name.show()
 		_tool_buttons.show()
+		$PopochiuFilter.show()
 		
+		# Fill info of Props, Hotspots, Walkable areas, Regions and Points
 		for t in _types:
 			for c in opened_room.call(_types[t].method):
 				var row_path := ''
 				
 				if c is Position2D:
 					var row: PopochiuObjectRow = _create_object_row(t, c.name)
 					_types[t].group.add(row)
+					
+					continue
+				elif c is PopochiuCharacter:
+					# Get the script_name of the character
+					var char_name: String =\
+					c.name.lstrip('Character').replace(' *', '')
```

Comment:
Learning attempt comment: what are you doing here?

Reply:
It removes the "Character" word starting the search from the left of `c.name`, and in the result `String` it removes the " *". As I write, I realize that I could have also done `.rstrip(' *')`. I'll change it.

Reply:
Yes, `rstrip()` can do the trick. I was curious why the character name has that asterisk at the end, but it's better for me to take the time to understand the context.